apps/frontend/src/app/components/header/header.component.scss

```diff
@@ -211,7 +211,13 @@
   }
 
   &.open {
-    &:hover,
+    &:hover .submenu,
+    .submenu {
+      @media screen and (max-width: 992px) {
+        display: block;
+        opacity: 1;
+      }
+    }
```

Comment:
The only thing the you should have changed is `&:hover` to `&:hover .submenu`. The comma is a selector list operator. You can read about it here: https://developer.mozilla.org/en-US/docs/Web/CSS/Selector_list